### PR TITLE
Node

### PR DESCRIPTION
**Pull request by dgreisen-cfpb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,26 @@ Prerequisites:
...
 1. `vagrant up`
 1. `vagrant ssh`
-1. `cd /vagrant`
-1. `./manage.py syncdb --noinput`
-1. `./manage.py runserver 0.0.0.0:8000`
+1. load data from github:
+  1. `cd /opt/kratos`
+  1. `icake -n devdesign import_from_gh`
+1. start the django server
+  1. `/vagrant/devops/manage.py runserver 0.0.0.0:8000`
```
Comment (m3brown) [inline README.md]: You added gunicorn to your requirements.txt.  Is that in use yet (along with something like supervisor), or is that still a work in progress hence the need to launch runserver in the manual steps?

**Comment by m3brown** [comment]
I didn't review the static files, but :+1: for everything else.

File: README.md (hunk 1 above)
Comment (dgreisen-cfpb) [inline README.md]: gunicorn is working. but I didn't know how to set it up to restart when it detects code change. so for dev, I use runserver.

**Comment by dgreisen-cfpb** [comment]
Mike and I reviewed FE code.

**Comment by himedlooff** [comment]
:+1:

**Pull request merged**